website/docs/tutorial/index.mdx

```diff
@@ -117,12 +117,24 @@ Now, let's create an `index.html` at the root of the project.
 Run the following command to build and serve the application locally.
 
 ```bash
-trunk serve --open
+trunk serve --open // remove option '--open' to not open your default browser
 ```
 
 Trunk will open your application in your default browser, watch the project directory and helpfully rebuild your
-application if you modify any source files. If you are curious, you can run `trunk help` and `trunk help <subcommand>`
-for more details on what is happening.
+application if you modify any source files.
+This will fail if the socket is being used by another application.
+By default server will listening at address '127.0.0.1' and port '8080' [localhost:8080](http://127.0.0.1:8080).
+To change it, create the following file and edit as needed:
```

Comment:
```suggestion
To change it, create the [Trunk config file](https://trunkrs.dev/configuration/#trunk-toml) and edit as needed:
```

Reply:
@hamza1311 suggested changes implemented in PR #3406.